<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrienListWidget</class>
 <widget class="QWidget" name="FrienListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Friends List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="friendRequestFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pendingRequestsButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QPushButton {
	border: 0px;
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:hover {
	border: 0px;
	background-color: rgba(255, 255, 255, 0);
	color: rgb(51, 98, 255);
}

QPushButton::pressed {
	border: 0px;
	background-color: rgba(255, 255, 255, 0);
	color: rgb(74, 122, 255);
}</string>
           </property>
           <property name="text">
            <string>x new friend request</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="friendListWidget">
        <property name="styleSheet">
         <string>QListView::item {
     min-height: 30px;
 }
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusEditor">
        <property name="font">
         <font>
          <family>Estrangelo Edessa</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
	background-color: qlineargradient(spread:pad, x1:0.006, y1:0, x2:0, y2:1, stop:0.142045 rgba(240, 246, 255, 255), stop:0.295455 rgba(226, 233, 239, 255), stop:0.744318 rgba(204, 210, 225, 255), stop:1 rgba(222, 227, 236, 255));
	border-radius: 0px;
	min-height: 25px;
	font: 10pt &quot;Estrangelo Edessa&quot;;
}

QComboBox::drop-down {
	subcontrol-position: center right;
	width: 25px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
}

QComboBox::down-arrow {
    image: url(:images/arrow_down_hover.png);
}

QComboBox QAbstractItemView {
	padding-top: 5px;
	padding-bottom: 5px;
	selection-background-color: rgb(222, 227, 236);
	selection-color: rgb(0,0,0);
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
